src/main/java/com/volunteer/enterprise/dao/IVolunteerServiceDAO.java

```diff
@@ -0,0 +1,15 @@
+package com.volunteer.enterprise.dao;
+
+import com.volunteer.enterprise.dto.Volunteer;
+import java.util.List;
+
+
+public interface IVolunteerServiceDAO {
+    Volunteer save(Volunteer volunteer) throws Exception;
+
+    List<Volunteer> fetchAll();
+
+    Volunteer fetch(int id);
+
+    void delete(int id);
+}
```

Comment:
Added the right package and refactored VolunteerServiceDAO to IVolunteerServiceDAO. The naming convention is important.